tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go

```diff
@@ -190,15 +192,15 @@ func (r *RKE1NodeDriverProvisioningTestSuite) TestProvisioningRKE1ClusterDynamic
 }
 
 func (r *RKE1NodeDriverProvisioningTestSuite) testProvisioningRKE1Cluster(client *rancher.Client, provider Provider, nodesAndRoles []nodepools.NodeRoles, kubeVersion, cni string) (*management.Cluster, error) {
+	nodeTemplateResp, err := provider.NodeTemplateFunc(client)
```

Comment:
Another nit, I think this is not a response but a returned value from a helper function.

Reply:
Good point, took off the `Resp` and just named it as `nodeTemplate`.